2016-2017_Starstruck/Module Test Code/FileIO_Test/include/main.h

```diff
@@ -44,6 +44,9 @@
 #define MAIN_H_
 
 #include <API.h>
+#include <AutonRecorder.h>
+#include <stdlib.h>
```

Comment:
I think stdlib is included in API.h, but not sure. string.h should be included in AutonRecorder.h to make sure dependencies are managed properly (if this is being imported into a new project, you don't want the user to have to worry about adding system imports that your module is dependent on)